src/commands/deployWorkspaceProject/getDefaultValues/getDefaultContextValues.ts

```diff
@@ -23,9 +23,13 @@ export async function getDefaultContextValues(context: ISubscriptionActionContex
     const { rootFolder, dockerfilePath } = await getWorkspaceProjectPaths(context);
     const settings: DeployWorkspaceProjectSettings = await getDeployWorkspaceProjectSettings(rootFolder);
 
+    setDeployWorkspaceProjectSettingsTelemetry(context, settings);
+
     if (triggerSettingsOverride(settings, item)) {
         // Tree item / settings conflict
+        context.telemetry.properties.triggeredSettingsOverride = 'true';
```

Comment:
I think it could be easier for us to query this property if it was formatted like:

`properties.settingsOverride = triggered|accepted`. We know if it was accepted, it was triggered. That way we can just query the one property (unless there's some reason we'd only query one property and not the other)